tests/test_secret_tools.py

```diff
@@ -36,7 +36,7 @@ def test_is_secret_ref():
     # if it is not a string it can't be a secret ref
     # so this checks that we are catching the TypeError
     assert not is_secret_ref(None)
-    assert not is_secret_ref(3)
+    assert not is_secret_ref("3")
```

Comment:
this assert is explicitly meant to check for non-string values; maybe do type: ignore instead

Reply:
Aha, ok, will change.

Reply:
Or, well, is this something that mypy should prevent from happening (in which case we wouldn't need to test it, really) ?

Reply:
We talked and I am going to fix it -- everything that calls this method has type anotations, and they all only pass in `str`s.  However, since the origin of the values being passed to this method is yelpsoa-configs YAMLs, we can't totally guarantee that it'll only get `str`s.  Maybe we can add more validation?  But in the meantime we'll keep this test.